### ISSUE TITLE
Remove HCT related code 

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-frontend#7317]
For code quality let's remove the HCT code, config  + docs.

**For QA:** Google Analytics events should work as intended. We should not see any "HCT" logs or events anymore (in the console or in the Telemetry tab in FF).

**Comment by kumar303** [comment]
HCT = Hybrid Content Telemetry

**Comment by ioanarusiczki** [comment]
@willdurand @muffinresearch 
I verified this with FF64 unbranded (Win10) but I need a confirmation that my testing was ok. 
This is what I've done: 
1. I used in the Browser Console 
 let hostURI = Services.io.newURI('https://discovery.addons-dev.allizom.org');
Services.perms.add(hostURI, 'hc_telemetry', Services.perms.ALLOW_ACTION); 
2. From about:addons I installed/removed a theme, an extension 

This is what the Broswer Console displays (indeed, no HCT in the Console)

![console and dynamic tab](https://user-images.githubusercontent.com/33448286/51191765-c997df00-18ed-11e9-921d-2e54f27bd4d4.png)

However, I've noticed some events in about:telemetry in the "parent" tab (selecting "dynamic" from filter will no longer display anything)

![parent tab - telemetry](https://user-images.githubusercontent.com/33448286/51191933-1e3b5a00-18ee-11e9-89a5-cdedbee3bb4e.png)
